manim/mobject/coordinate_systems.py

```diff
@@ -1084,6 +1085,69 @@ def construct(self):
             **kwargs,
         ).set_opacity(opacity=opacity)
 
+    def get_area(
+        self,
+        graph: "ParametricFunction",
+        x_range: Optional[Sequence[float]] = None,
+        color: Union[Color, Iterable[Color]] = [BLUE, GREEN],
+        opacity: float = 0.3,
+        bounded: "ParametricFunction" = None,
+        **kwargs,
+    ):
+        """Returns a :class:`~.Polygon` representing the area under the graph passed.
+
+        Examples
+        --------
+
+        .. manim:: GetAreaExample
+            :save_last_frame:
+
+            class GetAreaExample(Scene):
+                def construct(self):
+                    ax = Axes().add_coordinates()
+                    curve = ax.get_graph(lambda x: 2 * np.sin(x), color=DARK_BLUE)
+                    area = ax.get_area(
+                        curve,
+                        x_range=(PI / 2, 3 * PI / 2),
+                        color=(GREEN_B, GREEN_D),
+                        opacity=1,
+                    )
+
+                    self.add(ax, curve, area)
+
+        Parameters
+        ----------
+        graph
+            The graph/curve for which the area needs to be gotten.
+        x_range
+            The range of the minimum and maximum x-values of the area. ``x_range = [x_min, x_max]``.
+        color
+            The color of the area. Creates a gradient if a list of colors is provided.
+        opacity
+            The opacity of the area.
+        bounded
+            If a secondary :attr:`graph` is specified, encloses the area between the two curves.
+        kwargs
+            Additional parameters passed to :class:`~.Polygon`
+
+        Returns
+        -------
+        :class:`~.Polygon`
+            The :class:`~.Polygon` representing the area.
+        """
+        a, b = x_range
+        if bounded is None:
+            points = (
+                [self.c2p(a)]
+                + [p for p in graph.get_points() if a <= self.p2c(p)[0] <= b]
```

Comment:
You should use filter here instead of list comps

Reply:
I thought that as well, but at the end of the day it would be maybe a little less readable in my eyes.

Reply:
`filter` is not particularly Pythonic. If there is no good reason for using `filter` over the list comprehension, I'd rather keep this as-is.

Reply:
(I also think flake will wine about it, with the flake updates in an open pull that should be merged after the update goes out)

Reply:
Ok then, I see this is a topic of hot debate.

Reply:
Why so?

Reply:
https://stackoverflow.com/questions/3013449/list-comprehension-vs-lambda-filter